etc/github-develop-requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+https://github.com/modflowpy/pymake@master
```

Comment:
suggest naming this file `requirements.mf6.txt`, they are pulled selectively from the mf6 environment to run the mf6 test suite